docs/user-documentation/media.md

```diff
@@ -1,60 +1,84 @@
 Drupal 8 recognizes files (such as images, audio files, video files, etc.) but wraps each file in an intermediate structure called a
-"media" to allow us to attach fields to files. It is in a media's fields that we store information about the media's file, such as file
+"media" to allow us to attach fields to files. Drupal uses a media's fields to store information about the media's file, such as file
 size, width and height (for images), alt text (for images), creation date, and so on.
 
 !!! note "Compared to Islandora 7"
     In Islandora 7, this sort of technical metadata would have been stored in a single RELS-INT datastream.
-    In Islandora 8, each datastream holds its own technical metadata using media entities.
+    In Islandora 8, each datastream holds its technical metadata using an associated media entity.
+    
+Fedora will store the media's file as a binary resource and use the media's properties as the binary resource's description.
+
+For example, creating a media with a file named `test.jpg` (in November 2019) will create 
+a Fedora binary resource of the file accessible at `/fcrepo/rest/2019-11/test.jpg` 
+with the media's fields accessible at `/fcrepo/rest/2019-11/test.jpg/fcr:metadata`.
+
+
+## Media revisions
+
+The metadata associated with a file can be updated by clicking the _Edit_ tab while on the media's page and clicking **Save**.
+
+The _Create new revision_ checkbox is selected by default which will prompt Fedora to make a new version of the media
+before updating its resource record. A message can be added to the revision which is stored in Drupal but is not currently saved in Fedora.
+
+!!! note "Versioning binary resources is not currently supported"
+    Drupal does not like users replacing files with the same name. 
+
+_Something_ is not allowing us to replace binaries. So, Media metadata can be versioned, but adding a new file will create a new Fedora Binary/Resource pair. Need to explore _why_.
```

Comment:
Drupal doesn't really expect or want you to update file contents.  The upload form for media that I demo'd last week could be repurposed to provide ' replace datastream' type functionality through the UI.  Then we could trigger binary versions in fedora.  After that we can hide the file field from the edit form and force everyone to go through the new form.

Reply:
I thought I had fixed that block of text.... where did my change go? 😕

Reply:
Okay, it looks like the new bit I added to explain this (under "Do not replace a media's file") is there; I just accidentally left this piece behind.